pkgs/by-name/ar/argocd/package.nix

```
@@ -51,6 +51,7 @@ buildGoModule rec {
   postInstall = ''
```

```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

If you're touching completions anyway, this will fix cross-compilation (see https://nixos.org/manual/nixpkgs/unstable/#installshellfiles), e.g. `nix build .#pkgsCross.aarch64-multiplatform.argocd`. You will also need to add `stdenv` to the arguments at the top.

---

done! (although not manually tested yet)